sonic-net/sonic-buildimage
# Support for serdes platform library debian installation for Innovium SONiC image

Signed-off-by: vijayvyasm vijayvyasm@marvell.com

#### Why I did it

- Needed to upstream the Innovium serdes platform library debian and the necessary steps to install the debian on the host during installation of the new sonic-image binary on a platform.

#### How I did it

- I added support for the Innovium serdes platform library debian download and installation in platform/innovium/invm-sai.mk and platform/innovium/one-image.mk respectively

#### How to verify it

- Build a new SONiC image and install it in an Innovium supported Wistron platform, then verify that the new library is installed. Example - /usr/share/sonic/device/x86_64-wistron_sw_to3200k-r0/lib_ivm_serdes_pltfm.so

#### Which release branch to backport (provide reason below if selected)


Need to backport to 202111 since Customer requires this support on their 202111 branch.

#### Description for the changelog
Support for serdes platform library debian installation for Innovium SONiC image

#### Link to config_db schema for YANG module changes
Not Applicable

#### A picture of a cute animal (not mandatory but encouraged)
